flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/async/AsyncWaitOperatorFactory.java

```diff
@@ -65,20 +68,11 @@ public StreamOperator createStreamOperator(StreamTask containingTask, StreamConf
 				capacity,
 				outputMode,
 				mailboxExecutor);
+		asyncWaitOperator.setProcessingTimeService(processingTimeService);
```

Comment:
For `AsyncWaitOperatorFactory` and `CodeGenOperatorFactory`, we can move the `processingTimeService` parameter to the constructors. For `SimpleOperatorFactory`, it seems that we can't do this, because what is stored in `SimpleOperatorFactory` is already a serialized operator object, unlike the operators of `AsyncWaitOperatorFactory` and `CodeGenOperatorFactory`, which are constructed in runtime.